tests/components/homekit/test_type_media_players.py

```diff
@@ -342,6 +345,19 @@ async def test_media_player_television(hass, hk_driver, events, caplog):
     assert len(events) == 11
     assert events[-1].data[ATTR_VALUE] is None
 
+    events = []
+
+    def listener(event):
+        events.append(event)
+
+    hass.bus.async_listen(EVENT_HOMEKIT_TV_REMOTE_KEY_PRESSED, listener)
+
```

Comment:
We should test that an unknown key doesn't send another event.

```suggestion

    await hass.async_add_executor_job(acc.char_remote_key.client_update_value, 20)
    await hass.async_block_till_done()

```